lib/nexmo_developer/config/application.rb

```diff
@@ -38,6 +38,7 @@ class Application < Rails::Application
     config.docs_base_path = ENV.fetch('DOCS_BASE_PATH', '.')
     config.middleware.use ::ActionDispatch::Static, "#{Rails.configuration.docs_base_path}/public", index: 'index'
     config.oas_path = ENV.fetch('OAS_PATH', './_open_api/api_specs/definitions')
+    config.blog_path = ENV.fetch('BLOG_PATH', './_blog')
```

Comment:
can we have `_blog` as part of the BLOG_PATH? Will be counter-intuitive to set BLOG_PATH env variable to the parent.